cms/tests/page.py

```diff
@@ -112,6 +114,25 @@ def test_create_page_api(self):
         self.assertEqual(list(Title.objects.drafts().values_list('path', flat=True)), [u''])
         self.assertEqual(list(Title.objects.public().values_list('path', flat=True)), [u''])
 
+    def test_delete_page_no_template(self):
+        page_data = {
+            'title': 'root',
+            'slug': 'root',
+            'language': settings.LANGUAGES[0][0],
+            'template': 'nav_playground.html',
+
+        }
+        page = create_page(**page_data)
+        page.template = 'no_such_template.html'
+        signals.pre_save.disconnect(pre_save_page, sender=Page, dispatch_uid='cms_pre_save_page')
+        signals.post_save.disconnect(post_save_page, sender=Page, dispatch_uid='cms_post_save_page')
+        page.save(no_signals=True)
```

Comment:
It seems odd that we mute the signals in tests, but not in the real code. I don't get it. How is this a valid test?